### PR TITLE
Autotests: #2924 - test cases for functional groups and toolbar

### PR DESCRIPTION
**Pull request by AleksandraSmolianinova**
## How the feature works? / How did you fix the issue?
(Screenshots, videos, or GIFs, if applicable)

## Check list
- [ ] unit-tests written
- [x] e2e-tests written
- [ ] documentation updated
- [ ] PR name follows the pattern `#1234 – issue name`
- [x] branch name doesn't contain '#'
- [x] PR is linked with the issue
- [x] base branch (master or release/xx) is correct
- [ ] task status changed to "Code review"

### CODE REVIEW

**Review by MartaWilliams — COMMENTED** [review]
According to KB

Pull Request title must be exactly the same as the issue name and NEDD TO START with #task number. 
Example: '#1234 Improve search performance.

**Review by Zhirnoff — APPROVED** [review]

File: ketcher-autotests/tests/User-Interface/Toolbar/toolbar.spec.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+import {
+  selectTopPanelButton,
+  takeEditorScreenshot,
+  takeLeftToolbarScreenshot,
+  TopPanelButton,
+} from '@utils';
+
+test.describe('Open Ketcher', () => {
+  test.beforeEach(async ({ page }) => {
+    await page.goto('');
```
Comment (Nitvex) [inline ketcher-autotests/tests/User-Interface/Toolbar/toolbar.spec.ts:11]: Can we replace it with `waitForPageInit`?

File: ketcher-autotests/tests/User-Interface/Toolbar/toolbar.spec.ts [hunk 2]
```diff
@@ -0,0 +1,30 @@
+import { test } from '@playwright/test';
+import {
+  selectTopPanelButton,
+  takeEditorScreenshot,
+  takeLeftToolbarScreenshot,
+  TopPanelButton,
+} from '@utils';
+
+test.describe('Open Ketcher', () => {
```
Comment (Nitvex) [inline ketcher-autotests/tests/User-Interface/Toolbar/toolbar.spec.ts:9]: Could we add more meaningful text for describe?

**Review by Nitvex — CHANGES_REQUESTED** [review]

File: ketcher-autotests/tests/User-Interface/Toolbar/toolbar.spec.ts (hunk 1 above)
Comment (Zhirnoff) [reply ketcher-autotests/tests/User-Interface/Toolbar/toolbar.spec.ts:11]: Done:)

File: ketcher-autotests/tests/User-Interface/Toolbar/toolbar.spec.ts (hunk 2 above)
Comment (Zhirnoff) [reply ketcher-autotests/tests/User-Interface/Toolbar/toolbar.spec.ts:9]: Done:)

**Review by Nitvex — APPROVED** [review]

**Pull request merged**